apis/swagger.yml

```diff
@@ -1467,6 +1467,20 @@ paths:
             $ref: "#/definitions/NetworkDisconnect"
       tags: ["Network"]
 
+  /commit:
+    post:
+      summary: "Create a image from a container"
+      operationId: "Commit"
+      consumes: ["application/json"]
+      produces: ["application/json"]
+      responses:
+        201:
+          description: "The image was created successfully"
+          schema:
+            $ref: "#/definitions/ContainerCommitOptions"
```

Comment:
Does this make sense? I think it should be `ContainerCommitResp` in response rather than `ContainerCommitOptions`. WDYT?

Reply:
`ContainerCommitOptions` should be filled in the request body parameters, I think.

Like `ContainerCreateConfig` in request body parameter of the API endpoint.
```
  /containers/create:
    post:
      summary: "Create a container"
      consumes:
        - "application/json"
      produces:
        - "application/json"
      parameters:
        - name: "body"
          in: "body"
          description: "Container to create"
          schema:
            $ref: "#/definitions/ContainerCreateConfig"
          required: true
```

Reply:
update.